CONTRIBUTING.md

```diff
@@ -1,21 +1,33 @@
 
-### Communication
+## Ways to contribute
 
-If you have an idea or a question we'd love to hear from you! The easiest way to get in touch is by raising issues in the [medic Github repo](https://github.com/medic/medic/issues). You can also reach out on our community Slack channel. You can request access via [this _Community Health Toolkit_ page](https://communityhealthtoolkit.org/slack).
+### First time contributor?
+Issues labeled help wanted are a great place to start. Looking for other ways to help? You can also:
+* Improve our [documentation](#improving-our-documentation)
+* Find and mark duplicate issues
+* Try to reproduce issues and help with troubleshooting
+* Or [share a new idea](https://communityhealthtoolkit.org/forum) or [question](https://communityhealthtoolkit.org/forum) with us!
 
 ### Submitting code
+**Note:** We recommend you raise an issue on Github or start a conversation on our Community Forum about the change you want to make before you start on code.
 
-1. Before starting on code we recommend you raise an issue or message us on [Slack](https://communityhealthtoolkit.org/slack) to start a discussion about the change you want to make. Existing issues that have been identified as good for first time contributors are labeled with ["Help Wanted"](https://github.com/medic/medic/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22).
-2. When working on your submission read our [development workflow](https://github.com/medic/medic-docs/blob/master/development/workflow.md) to help understand how we work.
-3. When you're happy with the code be sure to run tests and precommit checks.
-4. Now submit a PR with a full description of the issue you're solving and if possible including a link to the GitHub issue.
-5. [Our CI](https://travis-ci.org/medic/) will automatically schedule a build. Keep an eye on the build to make sure it passes.
-6. Your PR will be reviewed by a Medic team member. Most PRs have at least one change requested before they're merged so don't be offended if your change doesn't get accepted on the first try!
+1. Read our [Development Workflow](https://github.com/medic/medic-docs/blob/master/development/workflow.md) to understand how we work, and review our [Code Style Guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md) before you begin.
+2. Before you submit a pull request, please make sure your contribution passes all tests. Test failures need to be addressed before we can merge your contribution.
+3. Provide detail about the issue you are solving in the pull request description. Note: If your pull request addresses a specific issue, use `address #[issue number]` or `closes #[issue number]` syntax in the pull request description as well.
```

Comment:
We don't use the `closes` pattern because GH automatically closes the issue when it's merged, which isn't always the right thing to do. We just list the issue number without prefix, eg: `medic/medic#[issue number]`

Reply:
That makes total sense! I revised, but could add in the example you gave above as well, if you think it's helpful.

Reply:
It's probably useful I think. It's also prepopulated as a PR template but we can include it here too.

Reply:
Done! I think.